dashboard/config/locales/scripts.en.yml

```diff
@@ -19155,48 +18879,30 @@ en:
             Welcome to the Internet:
               key: Welcome to the Internet
               name: Welcome to the Internet
-              description_student: ''
-              description_teacher: ''
             Building a Network:
               key: Building a Network
               name: Building a Network
-              description_student: ''
-              description_teacher: ''
             The Need for Addressing:
               key: The Need for Addressing
               name: The Need for Addressing
-              description_student: ''
-              description_teacher: ''
             Routers and Redundancy:
               key: Routers and Redundancy
               name: Routers and Redundancy
-              description_student: ''
-              description_teacher: ''
             Packets:
               key: Packets
               name: Packets
-              description_student: ''
-              description_teacher: ''
             DNS and HTTP:
               key: DNS and HTTP
               name: DNS and HTTP
-              description_student: ''
-              description_teacher: ''
             Project - Internet Dilemmas Part 1:
               key: Project - Internet Dilemmas Part 1
               name: Project - Internet Dilemmas Part 1
-              description_student: ''
-              description_teacher: ''
             Project - Internet Dilemmas Part 2:
               key: Project - Internet Dilemmas Part 2
               name: Project - Internet Dilemmas Part 2
-              description_student: ''
-              description_teacher: ''
             'Lesson 9: Assessment Day':
               key: 'Lesson 9: Assessment Day'
               name: 'Lesson 9: Assessment Day'
-              description_student: ''
-              description_teacher: ''
           lesson_groups:
             csp_unit2_2020:
```

Comment:
why is this named csp_unit2_2020 and not csp2-2021?

Reply:
It looks like that is the lesson group key